Meadowlark-js/src/metaed/MetaEdProjectMetadata.ts

```diff
@@ -38,8 +38,7 @@ export interface MetaEdProjectMetadata {
  * A hardcoded mapping of abbreviations to Ed-Fi Model npm package names
  */
 export function modelPackageFor(projectAbbreviation: string): string {
-  if (projectAbbreviation === Constants.version22) return 'ed-fi-model-2.2';
-  if (projectAbbreviation === Constants.version33b) return 'ed-fi-model-3.3b';
+  if (projectAbbreviation === Constants.version33b) return '@edfi/ed-fi-model-3.3b';
```

Comment:
I forgot that we have some hard-coding to the model version. 

In my Meadowlark prep ticket, it would be nice to add support for Data Model 3.1 and have instructions on switching. Powerful demonstration for next week. We can contemplate that in RND-222. I may need your help with that.

Swagger metadata would be wrong, since we currently have a static JSON file. But we can look past that detail.